Lavanya.C.Sharp/Set1Task4/Program.cs

```diff
@@ -0,0 +1,30 @@
+﻿
+int a = 15;
+int b = 10;
+decimal c;
+
+c = a + b;
+Console.WriteLine($"Sum of {a} and {b} is {c}");
+Print(++a, --b);
+
+c = a-b;
+Console.WriteLine($"Difference of {a} and {b} is {c}");
+Print(++a, --b);
+
+c = a*b;
+Console.WriteLine($"Product of {a} and {b} is {c}");
+Print(++a, --b);
+
+c = a/b;
+Console.WriteLine($"quotient of {a} and {b} is {c}");
+Print(++a, --b);
+
+c = a%b;
+Console.WriteLine($"Reaminder of {a} and {b} is {c}");
+Print(++a, --b);
+
+void Print(int a , int b)
```

Comment:
how about leveraging the method for the other console.writeline too?

Reply:
Moved all console.writeline to a single method but I have to use ref parameters